docs/index.rst

```diff
@@ -1,152 +1,36 @@
-:full-width:
-:hide-version-warning:
-:hide-pre-content:
-:hide-post-content:
-:hide-sidebar:
-:hide-secondary-sidebar:
-:landing:
+:orphan:
 
-.. title:: Welcome to ScyllaDB Documentation
+==================================
+ScyllaDB Open Source Documentation
+==================================
 
-.. hero-box::
-  :title: Welcome to ScyllaDB Documentation
-  :image: /_static/img/mascots/scylla-docs.svg
-  :search_box:
+.. meta::
+   :title: ScyllaDB Open Source Documentation
+   :description: ScyllaDB Open Source Documentation
+   :keywords: ScyllaDB Open Source, Scylla Open Source, ScyllaDB Open Source documentation, ScyllaDB support
 
-  New to ScyllaDB? Start `here <https://cloud.docs.scylladb.com/stable/scylladb-basics/>`_!
+About This User Guide
+-----------------------
 
-.. raw:: html
+ScyllaDB Open Source is a distributed NoSQL wide-column database for data-intensive apps that require 
+high performance and low latency.
 
-  <div class="landing__content landing__content">
+You can deploy ScyllaDB Open Source:
 
-.. raw:: html
+* Todo
```

Comment:
@annastuchlik Could you please provide the content for the new ScyllaDB Open Source index page?

Reply:
We've decided to open two separate PRs for creating the new homepage (step 1) and updating the contents of the existing one (step 2).